### PR TITLE
Issue 4904 - Fix various small issues

### PR DESCRIPTION
**Pull request by mreynolds389**
- Issue 4904 - Set the plugin identity before calling internal op.
- Issue 5232 - Remove --durable-txn from dsconf
- Issue 5195 - Improve error messages to state with settings are missing
- Issue 5169 - During LDIF import we log a debug message by default when we should
not
- Issue 5259 - dsctl --remove-all can be triggered with abbreviated arg
 name

relates: https://github.com/389ds/389-ds-base/issues/4904
relates: https://github.com/389ds/389-ds-base/issues/5232
relates: https://github.com/389ds/389-ds-base/issues/5195
relates: https://github.com/389ds/389-ds-base/issues/5169
relates: https://github.com/389ds/389-ds-base/issues/5259

### CODE REVIEW

**Review by Firstyear — APPROVED** [review]

**Review by droideck — APPROVED** [review]
LGTM!

**Review by tbordaz — APPROVED** [review]
LGTM

File: src/lib389/cli/dsctl [hunk 1]
```diff
@@ -33,7 +33,7 @@ from lib389.cli_base import (
     format_error_to_dict)
 from lib389._constants import DSRC_CONTAINER
 
-parser = argparse.ArgumentParser()
+parser = argparse.ArgumentParser(allow_abbrev=False)
```
Comment (progier389) [inline src/lib389/cli/dsctl:36]: I am afraid that this change will have unwanted side effect  (unable to pack the short options together) (for example  dsconf -vvvvv ... will have to be chamged to dsconf -v -v -v -v -v ... ) that may impact customer scripts ... 
IMHO we have better double check the options by ourself by checking that '--removal-all' is in sys.argv

**Review by progier389 — CHANGES_REQUESTED** [review]

File: src/lib389/cli/dsctl (hunk 1 above)
Comment (mreynolds389) [reply src/lib389/cli/dsctl:36]: I didn't think we used "-vvvvv", or any other arg option in that way.  I thought it was only -v.  I see no difference in the output:

[root@fedora cli]# dsconf -vvvvvv localhost config get | wc -l
362
[root@fedora cli]# dsconf -v localhost config get | wc -l
362

I don't mind changing the patch, but I don't "think" this is a concern.  Am I missing something?

Comment (progier389) [reply src/lib389/cli/dsctl:36]: Sorry for the noise, 
    finally there is no concern at all:   dsctl -jl is still working.
I just misread the Argparse documentation:
  In fact the short option issue was fixed in python 3.8 (I initially read (too quickly) the opposite):
Changed in version 3.8: In previous versions, allow_abbrev also disabled grouping of short flags such as -vv to mean -v -v.

**Review by progier389 — APPROVED** [review]
LGTM

File: src/lib389/cli/dsctl (hunk 1 above)
Comment (mreynolds389) [reply src/lib389/cli/dsctl:36]: All good, I appreciate the review, and apparently it really was a problem not that long ago.  Glad it's fixed now :-)

**Pull request merged**